GeneratedModels/MSGraphChatMessageHostedContent.h

```diff
@@ -3,9 +3,9 @@
 
 
 
-#import "MSGraphEntity.h"
+#import "MSGraphTeamworkHostedContent.h"
 
-@interface MSGraphChatMessageHostedContent : MSGraphEntity
+@interface MSGraphChatMessageHostedContent : MSGraphTeamworkHostedContent
```

Comment:
this is the only change I saw that is potentially breaking.

Reply:
Yeah, technically breaking for scenarios where client took a dependency on the fact that the base was named MSGraphTeamworkHostedContent.